.config/dotnet-tools.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": 1,
+  "isRoot": true,
+  "tools": {
+    "dotnet-serve": {
+      "version": "1.5.0",
```

Comment:
You updated the tool version to 1.7.125 in 'master'. Is that needed here too?

Reply:
Hmm, maybe. The test uses a fixed port now, which wasn't a problem in the wasm branch since we had so few template tests. I'll investigate.